README.md

```diff
@@ -497,9 +497,7 @@ Instead, you can ignore notices based on some condition.
 
 ```ruby
 Airbrake.add_filter do |notice|
-  if notice[:errors].any? { |error| error[:type] == 'StandardError' }
-    notice.ignore!
-  end
+  notice.ignore! if notice.exception.kind_of? StandardError
```

Comment:
Please use brackets around `.kind_of?(StandardError)`. We prefer this style in this project.